student/views/info.py

```
@@ -0,0 +1,61 @@
+
+            return Response(responseSerializer.data, status=status.HTTP_200_OK)
+
+        except (ClientException, ScraperException) as e:
+            return Response(
+                {
+                    "message": e.type.message,
+                },
+                status=e.type.httpStatus,
+            )
+
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)